crates/rccheck/src/tests/ed25519_remote_cert.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{ed25519_certgen::Ed25519, test_utils::cert_bytes_to_spki_bytes, *};
+use rcgen::generate_simple_self_signed;
+use rustls::{client::ServerCertVerifier, server::ClientCertVerifier};
+
+#[test]
+fn rc_gen_self_signed_dalek() {
```

Comment:
Please check this out and ignore everything else in the PR - is this something that seems viable to do?

Reply:
Yes, the script at a high-level looks plausible.